### PR TITLE
Documentation patch

### PR DESCRIPTION
**Pull request by ShohamBit**
1. Explain what the PR does:
- fixing the configuration docs of tracee
- added things to the global config 
- fixed docs issues
- added the copy conntent to mkdocs

2. Explain how to test it

view the section of:
Events in built in and extra events:

    security_settime64 that wasnt working
    security_path_notify wasnt exisit
    set_fs_pwd wasnt exisit
    tracee_info wasnt exisit

Events in built in and security events:

    format wasnt exist

CLI flags in scope:

    removed the option of using wildcard because it doesn't work with tracee

tutorials:
the link to Kubernetes guide didnt work

Overview file in the docs:

    section the Tutorials ,Docs and Contributing wasnt correct
    the macfaq was in () insted of []
    contribution documentation was with out the .md

you can view less importent things in the files changed of the git hub pr

### CODE REVIEW

File: docs/docs/install/config/cli.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ To use the `--config` flag, you need to provide the path to the configuration fi
 ```console
 tracee --config /path/to/tracee-config.yaml
 ```
+__NOTE:__ you can find an example config file in [here](/examples/config/)
```
Comment (geyslan) [inline docs/docs/install/config/cli.md:12]:
```suggestion
__NOTE:__ you can find an example config file in [here](/examples/config/).
```

File: docs/docs/install/config/index.md [hunk 2]
```diff
@@ -16,6 +18,9 @@ proctree:
     cache:
         process: 8192
         thread: 4096
+    cache-ttl:
+        process: 60
+        thread: 60
```
Comment (geyslan) [inline docs/docs/install/config/index.md:23]: This was removed already.

File: docs/docs/install/config/index.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 # Configuring Tracee
 
 Tracee has many different options and settings that control how Tracee operates. 
-This section presents available configuration options. To learn about how to apply configuration to Tracee, please refer to the [CLI](./cli.md) or [Kubernetes](./kubernetes.md) specific guides.
 
 A complete config file with all available options can be found [here](https://github.com/aquasecurity/tracee/blob/main/examples/config/global_config.yaml). Most of the options are documented in different sections in the documentation.
+To learn about all available configuration options please see [this document](https://github.com/aquasecurity/tracee/blob/main/examples/config/global_config.yaml).
```
Comment (geyslan) [inline docs/docs/install/config/index.md:6]:
```suggestion
To learn about all available configuration options please see [this config example](https://github.com/aquasecurity/tracee/blob/main/examples/config/global_config.yaml).
```

**Review by geyslan — COMMENTED** [review]
I did the first pass-through review (skimming).

**Comment by ShohamBit** [comment]
thank you @geyslan, do you think i should add back the part about the wildcard  or to wait for a respond from @NDStrahilevitz

File: docs/docs/policies/rules.md [hunk 4]
```diff
@@ -1,72 +1,46 @@
 # Rules
 
-Rules are part of the Tracee Policy, which defines which events to trace. The events that are part of a specific policy are recorded in the `rules` section of the Tracee Policy. It is possible to define multiple events within each policy. The [events](../events/index.md) section provides further information on the type of events that Tracee can track.
+Rules are part of the Tracee Policy, which defines which events to trace.
+The events that are part of a specific policy are recorded in the `rules` section of the Tracee Policy. 
+It is possible to define multiple events within each policy.`rules` section include:
```
[the remark this answers is not in the document]
Comment (ShohamBit) [reply docs/docs/policies/rules.md:5]: what did you not understand, basicly its the same as before

File: docs/docs/install/config/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ShohamBit) [reply docs/docs/install/config/index.md:6]: oh i understand i will fix

File: docs/docs/flags/scope.1.md [hunk 5]
```diff
@@ -49,9 +49,7 @@ Available for the following string fields:
 - container: Select events from specific container IDs.
 - executable: Select events based on the executable path.
 
-Strings can be compared as a prefix if ending with '\*', or as a suffix if starting with '\*'.
 
-NOTE: Expressions containing '\*' token must be escaped!
```
[the remark this answers is not in the document]
Comment (ShohamBit) [reply docs/docs/flags/scope.1.md:54]: ok will do

File: docs/docs/install/config/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ShohamBit) [reply docs/docs/install/config/index.md:23]: oh thats a big cach

File: docs/docs/flags/scope.1.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (NDStrahilevitz) [reply docs/docs/flags/scope.1.md]: IIRC string filters can currently filter by prefix implicitly. I do recall that there is a wildcardish behavior though. Need to check...
About events, yes, wildcards work in argument and context filters. In the future when @rscampos implements kernel filters we will need to specify the differences (as I assume that include filters such as `*blahblah*` will not work in kernel filters).

**Comment by geyslan** [comment]
> thank you @geyslan, do you think i should add back the part about the wildcard or to wait for a respond from @NDStrahilevitz

Don't mention it 👍🏼. He already answered.

File: docs/docs/policies/rules.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (geyslan) [reply docs/docs/policies/rules.md]: Ah, it was probably due to the agglutinated end part: 

"policy.`rules` section include:"

File: docs/docs/flags/scope.1.md [hunk 6]
```diff
@@ -53,6 +53,8 @@ Strings can be compared as a prefix if ending with '\*', or as a suffix if start
 
 NOTE: Expressions containing '\*' token must be escaped!
 
+you can learn more about the wildcard in the [`event` section](./events.1.md)
```
Comment (ShohamBit) [inline docs/docs/flags/scope.1.md:56]:
```suggestion
you can learn more about the wildcard in the [`event` section](./events.1.md).
```

File: docs/docs/policies/rules.md [hunk 7]
```diff
@@ -1,72 +1,50 @@
 # Rules
 
-Rules are part of the Tracee Policy, which defines which events to trace. The events that are part of a specific policy are recorded in the `rules` section of the Tracee Policy. It is possible to define multiple events within each policy. The [events](../events/index.md) section provides further information on the type of events that Tracee can track.
+Rules are part of the Tracee Policy, `rules` let you define which events to trace.
 
-Below are several examples on configuring events in the Tracee Policy.
+`rules` have 2 sections: 
+- events: let you define which events you wan to trace
+- filters: enable you to refine the policy's scope
```
Comment (ShohamBit) [inline docs/docs/policies/rules.md:7]:
```suggestion
- events: let you define which events you want to trace.
- filters: enable you to refine the policy's scope.
```

File: docs/docs/policies/rules.md [hunk 8]
```diff
@@ -1,72 +1,50 @@
...
-	scope:
-	    - global
-	rules:
-	    event: open
-```
-
-The name of the syscall is going to be the name of the event.
-
-**[Network Events](../events/builtin/network/index.md)**
+Tracee support many kind of events to tracee you can find which events you can trace in [Events section](/docs/docs/events/index.md)
```
Comment (ShohamBit) [inline docs/docs/policies/rules.md:9]:
```suggestion
Tracee support many kind of events to tracee you can find which events you can trace in [Events section](/docs/docs/events/index.md).
```

File: docs/docs/policies/rules.md [hunk 9]
```diff
@@ -1,72 +1,50 @@
...
+		- uid=1000
 ```
+## Events
 
-**[A behavioural Signature](../events/builtin/signatures/index.md)**
+### Type of Events
+The value of an `event` it's the event name from the supported events.
 
-To specify one of the behavioral signatures as an event, use the name of the signature from the table in the documentation as the event name:
+For example: `syscall` event would be the `syscall` name.
```
Comment (ShohamBit) [inline docs/docs/policies/rules.md:34]:
```suggestion
For example: `syscall` event would be the `syscall` event name.
```

File: docs/docs/policies/rules.md [hunk 10]
```diff
@@ -1,72 +1,50 @@
...
 
-```bash
-spec:
-	scope:
-	    - global
-	rules:
-	    event: anti_debugging
-```
+The [events](../events/index.md) section provides further information on the type of events that Tracee can track.
```
Comment (ShohamBit) [inline docs/docs/policies/rules.md:36]:
```suggestion
The [events](../events/index.md) section provides further information on the type of events that Tracee can trace.
```

**Review by itaysk — APPROVED** [review]

**Comment by yanivagman** [comment]
Why did you rename scope filters to process filters?
There are two reasons to keep calling it scope filters:
1. In the rule, we are further filtering the scope given in the policy
2. This is more intuitive for the user since both policy scope and rule scope filters are filtering the same fields
3. In the future, we want to extend the rule scope filters to also include container/pod related fields - then the term "process filters" will be incorrect

**Pull request merged**